lib/github/ldap.rb

```diff
@@ -100,6 +100,9 @@ def initialize(options = {})
 
       # enables instrumenting queries
       @instrumentation_service = options[:instrumentation_service]
+
+      # active directory forest
+      @forest = get_domain_forest(options[:search_forest])
```

Comment:
GitHub Enterprise instantiates a global strategy. As such, we need to provide an interface to refresh the list of domain controllers so that a configuration update does not require a restart of the appliance.